LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Lars van Rhijn
+Copyright (c) 2020 Lars van Rhijn, Job Doesburg
```

Comment:
Oehh I did not know GitHub rendered your username in the License file, nice spot!